src/common/options/global.yaml.in

```diff
@@ -3980,9 +3980,13 @@ options:
   desc: number of PGs for new pools
   fmt_desc: The default number of placement groups for a pool. The default
     value is the same as ``pg_num`` with ``mkpool``.
+  long_desc: With default value of `osd_pool_default_pg_autoscale_mode` being 
+    `on` the number of PGs for new pools will start out with 1 pg, unless the 
+    user specifies the pg_num.
   default: 32
   services:
   - mon
+  see_also: osd_pool_default_pg_autoscale_mode
```

Comment:
```suggestion
  see_also:
  - osd_pool_default_pg_autoscale_mode
```

please note, the value of the `see_also` field should be a list not a single string.

that's why this change fails to pass the validator of options:
```
Non-existent see-also key 'o' on option 'osd_pool_default_pg_num'
../src/common/config.cc: In function 'void md_config_t::validate_schema()' thread 7fcfdf9e0c00 time 2021-04-21T15:09:50.333944+0000
../src/common/config.cc: 248: ceph_abort_msg("abort() called")
 ceph version Development (no_version) quincy (dev)
 1: (ceph::__ceph_abort(char const*, int, char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)+0xfa) [0x7fcfe1b4f7c0]
 2: (md_config_t::validate_schema()+0x202) [0x7fcfe1bf43d6]
 3: (md_config_t::md_config_t(ConfigValues&, ConfigTracker const&, bool)+0xf289) [0x7fcfe1bf3acb]
 4: (ceph::common::ConfigProxy::ConfigProxy(bool)+0x6e) [0x7fcfe1b68596]
 5: (ceph::common::CephContext::CephContext(unsigned int, code_environment_t, int)+0x63) [0x7fcfe1b620d5]
 6: (common_preinit(CephInitParameters const&, code_environment_t, int)+0xed) [0x7fcfe1be233c]
 7: (global_pre_init(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > const*, std::vector<char const*, std::allocator<char const*> >&, unsigned int, code_environment_t, int)+0xe2) [0x55d8e81a9bec]
 8: (global_init(std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > const*, std::vector<char const*, std::allocator<char const*> >&, unsigned int, code_environment_t, int, bool)+0xb9) [0x55d8e81aa1c5]
 9: main()
 10: __libc_start_main()
 11: _start()
```

Reply:
Make sense, thank you!